pkg/model/components/kubelet.go

```diff
@@ -208,6 +195,9 @@ func (b *KubeletOptionsBuilder) BuildOptions(o interface{}) error {
 		if _, found := clusterSpec.Kubelet.FeatureGates["CSIMigrationAWS"]; !found {
 			clusterSpec.Kubelet.FeatureGates["CSIMigrationAWS"] = "true"
 		}
+		if _, found := clusterSpec.Kubelet.FeatureGates["CSIMigrationAWSComplete"]; !found {
+			clusterSpec.Kubelet.FeatureGates["CSIMigrationAWSComplete"] = "true"
+		}
```

Comment:
Can you rebase this against master? If not you may get into conflicts.

Reply:
Fixed